contracts/0.8.25/Accounting.sol

```diff
@@ -88,15 +88,23 @@ contract Accounting is VaultHub {
     ILido public immutable LIDO;
 
     constructor(
-        address _admin,
         ILidoLocator _lidoLocator,
         ILido _lido,
         address _treasury
-    ) VaultHub(_admin, _lido, _treasury) {
+    ) VaultHub(_lido, _treasury) {
         LIDO_LOCATOR = _lidoLocator;
         LIDO = _lido;
     }
 
+    function initialize(address _admin) external initializer {
+        if (_admin == address(0)) revert ZeroArgument("_admin");
+
+        __AccessControlEnumerable_init();
```

Comment:
Do we really need this?

Anyway, maybe move it to __VaultHub_init as Accounting does nothing with access control?
As well as the initial role grant?

Reply:
agree